docs/templates/layout.html

```diff
@@ -108,3 +113,22 @@
     }
   </style>
 {% endblock %}
+
+{% block footer %}
+{{ super() }}
+<div class="footer">This page uses <a href="https://analytics.google.com/">
+Google Analytics</a> to collect statistics. You can disable it by blocking
+the JavaScript coming from www.google-analytics.com. Check our
+<a href="privacy_notice.html">Privacy Policy</a>
```

Comment:
Does it work on subpages? 
For example: 
On this page: ``https://airflow.readthedocs.io/en/latest/howto/operator/gcp/natural_language.html``
This link will point to: ``https://airflow.readthedocs.io/en/latest/howto/operator/gcp/privay_notice.html``

Reply:
You are right. it won't work. Can you fix that, please?

Reply:
I am happy that I can do it.